### DISCUSSION TITLE
Python implementation

### DISCUSSION THREAD

**Discussion by akshayjshah** [connectrpc/.github#6] [Polls]
Would you or your company use a Connect implementation in Python? If yes, please leave a comment and tell us more about what you'd like: synchronous or asynchronous, important framework integrations (FastAPI? Django? Flask?), and anything else important.

**Comment by tinti** [comment]
We use it with AWS SAM and Lambda. Not only Connect but GRPC allows us to validate and parse incoming requests with ease. Our use case is only synchronous.

**Comment by cshubhamrao** [comment]
FastAPI integration with [betterproto](https://github.com/danielgtaylor/python-betterproto) like pydantic models would be awesome

**Comment by mattrobenolt** [comment]
Would like to toss up, I have a WIP implementation of this.

https://github.com/mattrobenolt/connect-python

It's very minimal documentation and whatnot for now. I'm going to get this moved over into the `connectrpc` org and start getting this more ready for others to poke at.

**Comment by mattrobenolt** [comment]
I have tossed up an RFC: https://github.com/connectrpc/connectrpc.com/pull/71

Would like to get some initial feedback.

**Comment by patrikdevlin** [comment]
We have some services written in python because they require specific python-based tooling, but still need to talk to the rest of our microservices. It would be great for those to also be able to leverage the generated sdk and connect.

Our use-case is mainly synchronous and framework isn't super important, but we use fastapi in some other places

**Comment by antspy** [comment]
Hi, Is this still on the roadmap? Is python support something that you think might be added? 

if not, what is the recommended approach to contact a python gRPC backend from the browser? Can I use connectRPC typescript from the browser and connect to a python gRPC backend ?

**Comment by chrispine** [comment]
https://github.com/connectrpc/connect-python
